bigquery_etl/glean_usage/templates/migrate_baseline_clients_daily_v1.sql

```diff
@@ -0,0 +1,14 @@
+{{ header }}
+
+SELECT
```

Comment:
I don't see clustering changes captured anywhere in this script.

I think the easiest thing to do may be to change these from queries with a destination table into CTAS statements:

```
CREATE OR REPLACE TABLE
  `{{ daily_table }}`
PARTITION BY submission_date
CLUSTER BY is_new_profile, normalized_channel, sample_id
AS
...
``` 

That would allow us to do the backfill and redefine the clustering in one operation. Otherwise, we'd need to update the schema to include the field and update the clustering configuration first before running these queries.

Reply:
I didn't anticipate clustering changes, though this makes sense in order for the queries on the is_new_profile column efficient. We actually don't need to update the schema to run these queries (I tested this in the backill-8 project), but updating the clustering configuration will require the changes you're proposing.

Reply:
This isn't necessary for the last seen table, is it?

Reply:
> This isn't necessary for the last seen table, is it?

I had been assuming we'd make the clustering change for last_seen, but I suppose it's not too impactful. The intent of the clustering is to enable an efficient `clients_first_seen` view, which only requires the daily table.

Reply:
It looks like we may have to set the destination into the backill project for these tables from the shared-prod project, and then copy the results back into shared prod. I ran a variant of the above DDL and got the following error:

```
google.api_core.exceptions.BadRequest: 400 POST https://bigquery.googleapis.com/bigquery/v2/projects/moz-fx-data-backfill-8/jobs?prettyPrint=false: Cannot replace a table with a different partitioning spec. Instead, DROP the table, and then recreate it. New partitioning spec is interval(type:day,field:submission_date) clustering(is_new_profile,normalized_channel,sample_id) and existing spec is interval(type:day,field:submission_date) clustering(normalized_channel,sample_id)
```

Reply:
> we may have to set the destination into the backill project for these tables from the shared-prod project, and then copy the results back into shared prod

That sounds like a good way forward. I had forgotten about this limitation of CREATE OR REPLACE.